src/sentry/static/sentry/app/views/settings/components/forms/select2Field.jsx

```diff
@@ -1,9 +1,24 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import jQuery from 'jquery';
+import classnames from 'classnames';
+import {css} from 'react-emotion';
 
 import InputField from './inputField';
 
+const formControlSmall = css`
```

Comment:
this is kind of ugly but it allows me to write code here without trying to completely port over form-control.